client/apps/edit/components/content2/test/index.test.coffee

```diff
@@ -1,89 +1,93 @@
-_ = require 'underscore'
-benv = require 'benv'
-sinon = require 'sinon'
-{ resolve } = require 'path'
-moment = require 'moment'
-fixtures = require '../../../../../../test/helpers/fixtures.coffee'
-Article = require '../../../../../models/article.coffee'
-React = require 'react'
-ReactDOM = require 'react-dom'
-ReactTestUtils = require 'react-addons-test-utils'
-r =
-  find: ReactTestUtils.findRenderedDOMComponentWithClass
-  simulate: ReactTestUtils.Simulate
+# _ = require 'underscore'
```

Comment:
Don't have the patience to figure out how to test non-jsx code... When y'all are done playing around with redux feel free to remove from sections i added it to and uncomment below test